pkg/bid/bid.go

```diff
@@ -0,0 +1,149 @@
+package bid
+
+import (
+	"fmt"
+	"strings"
+
+	v2 "github.com/conductorone/baton-sdk/pb/c1/connector/v2"
+)
+
+const (
+	BidPrefix            = "bid:"
+	ResourceBidPrefix    = BidPrefix + "r:"
+	EntitlementBidPrefix = BidPrefix + "e:"
+	GrantBidPrefix       = BidPrefix + "g:"
+)
+
+func ToBid(b interface{}) string {
```

Comment:
I feel like there's a way to be more strict with the typing here.

At the very least these could/should accept a 'proto.Message'.